renderer/webtorrent.js

```diff
@@ -244,7 +244,6 @@ function getTorrentProgress () {
 }
 
 function startServer (infoHash, index) {
-  if (server) return
```

Comment:
(The way it was before could actually leak servers. If you get to `startServer` twice on torrents that aren't ready, then both will pass the `if (server)` check, wait for their respective torrents to become ready, then start two servers, leaking one of them.)